chains/container-chains/runtime-templates/frontier/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -35,14 +35,14 @@
 // *
 // --chain=dev
 // --steps
-// 50
+// 16
 // --repeat
-// 20
+// 1
 // --template=./benchmarking/frame-weight-runtime-template-xcm.hbs
 // --json-file
 // raw.json
 // --output
-// tmp/frontier_template_weights/pallet_xcm_benchmarks_generic.rs
+// tmp/dancebox_weights/pallet_xcm_benchmarks_generic.rs
```

Comment:
this looks like cheating

Reply:
I only forgot to change the output folder, the benchmark is ok